app/api/service/user/__init__.py

```diff
@@ -0,0 +1,124 @@
+from flask import jsonify, current_app, request, Response
+from app.models.models import User
+import uuid
+from werkzeug.security import generate_password_hash
+import jwt
+from app.api.serializers.user import UserSchema
+from app.shortcuts import dbsession
+from app.api.decorators.token import token_required
+
+
+class UserService:
+
+    def __init__(self):
+        self.name = 'users'
+
+    @staticmethod
+    @token_required
+    def list(current_user):
+
+        if not current_user.admin:
+            return jsonify({'message': 'permission denied'})
+
+        users = User.query.all()
+
+        if not users:
+            return jsonify({'message': "there is no users yet"})
+
+        user_schema = UserSchema(many=True)
+
+        user_result = user_schema.dump(users).data
+
+        return jsonify({"users": user_result})
+
+    @staticmethod
+    @token_required
+    def one(current_user, id_):
+
+        if not current_user.id == int(id_) and not current_user.admin:
+            return jsonify({'message': 'permission denied'})
+
+        user = User.query.filter_by(id=int(id_)).first()
+
+        if not user:
+            return jsonify({'message': "there is no users"})
+
+        user_schema = UserSchema()
```

Comment:
`Schema` is for presentation level as a rule (i.e. the Controller one, i.e. the one for Flask View-s, not the Service).
I'd hand over the `user` (and other service objects throughout your code) from Service to View as-is, and would use `Schema` in the View only.